neofetch

```diff
@@ -4398,6 +4400,12 @@ display_image() {
             "$image"
         ;;
 
+        "viu")
+            viu \
+                -t -w ""$((width / font_width))"" -h "$((height / font_height))" \
```

Comment:
`""$((width / font_width))""` should be `"$((width / font_width))"`

Reply:
fixed